mkdocs/utils/ghp_import.py

```diff
@@ -167,7 +167,10 @@ def ghp_import(directory, message, remote='origin', branch='gh-pages'):
 
     run_import(directory, branch, message, nojekyll)
 
-    proc = sp.Popen(['git', 'push', remote, branch],
+    proc = sp.Popen(['git', 'push', remote, branch, '--force'],
```

Comment:
ghp_import leaves this as an option for the user. If we are going to add it, so should we. I understand the argument that your history gets hoses anyway, but that same argument could be used for ghp_import (we just repeat their warning) and they felt the need to make it optional. I'm inclined to follow their lead.

Maybe we should just update our vendored copy to a more recent version of ghp_import.